runtime.txt

```diff
@@ -0,0 +1,2 @@
+python-3.8
+# Specifies the python version that dependabot should use.
```

Comment:
Just FYI, I'm planning to try out migrating this repo to Poetry sometime soon, which should obviate the need for this (since Poetry solves for all Python versions and can deal with different constraints for each). OK to merge this now but might become unnecessary later.

Reply:
Yea, I was just looking to get in workarounds to avoid dependabot trying to update things that would fail CI.